recipe/meta.yaml

```
@@ -23,23 +23,22 @@ requirements:
     - setuptools
   run:
     - python
-    - girder-client
-    - pynwb >=1.0.3,!=1.1.0
     - appdirs
     - click
     - click-didyoumean
-    - email_validator
+    - dandischema
```

may be 
```suggestion
    - dandischema >= 0.2.3,<0.3
```
to reflect our requirement in setup.cfg?

---

Done.